src/Polly/Polly.csproj

```diff
@@ -1,50 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.1;netstandard2.0;net461;net472</TargetFrameworks>
-    <Version>7.2.4</Version>
-    <AssemblyVersion>7.0.0.0</AssemblyVersion>
-    <FileVersion>7.2.4.0</FileVersion>
-    <InformationalVersion>7.2.4.0</InformationalVersion>
-    <Company>App vNext</Company>
-    <Copyright>Copyright (c) $([System.DateTime]::Now.ToString(yyyy)), App vNext</Copyright>
-    <Description>Polly is a library that allows developers to express resilience and transient fault handling policies such as Retry, Circuit Breaker, Timeout, Bulkhead Isolation, and Fallback in a fluent and thread-safe manner.</Description>
-    <DefaultLanguage>en-US</DefaultLanguage>
-    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Authors>Michael Wolfenden, App vNext</Authors>
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources> <!-- EmbedUntrackedSources for deterministic build -->
-    <ContinuousIntegrationBuild Condition=" '$(CI)' != '' ">true</ContinuousIntegrationBuild>
-    <Deterministic>true</Deterministic>
+    <TargetFrameworks>netstandard2.0;net461;net472</TargetFrameworks>
```

Comment:
Did we discuss dropping `net461` in our last meeting? I can't remember.

Reply:
The `net461` is out of support, I think we can support the following frameworks:

- `net462`: still is supported
- `nestandard2.0`
- net6.0
- net7.0

The core frameworks bellow `net6.0` are out of support and are covered by  `nestandard2.0`